### ISSUE TITLE
Add a final "Proceed to publication" checkbox for reviewers

### ISSUE THREAD

**Issue by dpsanders** [openjournals/joss#829]
It often seems unclear exactly when a reviewer has finished their review.

Maybe it would useful to add a final checkbox along the lines of "I'm satisfied with the current status and happy for the submission to proceed to publication in JOSS".

**Comment by kyleniemeyer** [comment]
Normally I feel like this is (or should be) done via an explicit comment from the reviewer

**Comment by arfon** [comment]
I agree we want an explicit comment from the reviewer. Perhaps we should add something about this to the editor docs?

**Comment by jni** [comment]
Not just editor but also reviewers. Possibly rather than a check box, there can be a bullet point at the end of the checklist saying, "congratulations and thank you for making it all the way down the list! If you are satisfied with the paper and software, please write a comment in the conversation below confirming that you are happy for the paper to proceed to publication."

Or something along those lines?
